.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I'd argue that this is not really needed anymore?
You can pretty safely use es6 with node and restrict support to to NodeJS 6+ see https://github.com/nodejs/Release

Reply:
Alternatively you could give TypeScript a shot, it does everything babel does, and more!